### PR TITLE
Fix #4751: No name in "Reviews This Month" from Reviewer Tools for an user without display name set

### LINKED ISSUE #4751: No name in "Reviews This Month" from Reviewer Tools for an user without display name set
Steps to reproduce:
1. Go to Account Settings and delete your Display Name - https://addons.allizom.org/en-US/firefox/users/edit
2. Go to Reviewer Tools homepage and observe "Reviews This Month" section

Expected results:
Reviewer name and points are listed.

Actual results:
No name, just points.

Notes/Issues:
- I have a username set but is not used in that section.
Verified on FF51(Win 7). Issue is reproducing on AMO-stage and -dev.
Screenshot for this issue:

### PR DESCRIPTION
**Pull request by raajitr**
Fix #4751

i have changed the display_name to name to fallback if display name is empty.
But when i run "tests editors/test_view.py::test_stats_total" fails, the cols.eq(0).text() is returning a blank string (”).

### CODE REVIEW

**Comment by wagnerand** [comment]
Please add "Fix #4751" to your first comment.

Those tests fail because `row` does not have a property `name`. `row` is not a `user` object, it is an element in `reviews_monthly` and `reviews_total`, which are set in `src/olympia/editors/views.py#148`.

**Comment by raajitr** [comment]
Fix #4751 
Please let me know if any changes has to be made.

File: src/olympia/editors/templates/editors/home.html [hunk 1]
```diff
@@ -58,7 +58,11 @@
             <table>
               {% for row in reviews_total: %}
               <tr>
+                {% if row['user__display_name'] != '' %}
                 <td>{{ row['user__display_name'] }}</td>
+                {% else %}
+                <td>{{ row['user__username'] }}</td>
+                {% endif %}
```
Comment (wagnerand) [inline src/olympia/editors/templates/editors/home.html]: Could you try `{{ row['user__display_name'] or row['user__username'] }}` instead?

**Comment by caitmuenster** [comment]
Thanks so much, @raajitr! Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#April_2017), and I just vouched for your profile on [mozillians.org](https://mozillians.org/en-US/u/raajitr/). :) 

Welcome onboard! I look forward to seeing you around!

**Pull request merged**